src/screens/LocationPage/LocationPage.tsx

```diff
@@ -7,41 +7,33 @@ import MiniMap from 'components/MiniMap';
 import EnsureSharingIdInUrl from 'components/EnsureSharingIdInUrl';
 import ChartsHolder from 'components/LocationPage/ChartsHolder';
 import { LoadingScreen } from './LocationPage.style';
-import { useProjections } from 'common/utils/model';
+import { useProjectionsFromRegion } from 'common/utils/model';
 import { getPageTitle, getPageDescription } from './utils';
-import {
-  getCountyByUrlName,
-  getStateByUrlName,
-  getCanonicalUrl,
-} from 'common/locations';
+import { findCountyByFips, getCanonicalUrl } from 'common/locations';
+import { getStateCode, Region } from 'common/regions';
+import { assert } from 'common/utils';
 
-function LocationPage({ region }) {
-  let { stateId, countyId, chartId } = useParams();
+interface LocationPageProps {
+  region: Region;
+}
+
+function LocationPage({ region }: LocationPageProps) {
```

Comment:
Assigning the `React.FC` type to function components is slightly better than using a function with implicit return types because some components only accept children with the `React.FC` type. Not a big deal though!

```tsx
const LocationPage: React.FC<{LocationPageProps}> = ({}) => {

  return ();
}
```